test/unit/asrouter/ASRouterTargeting.test.js

```diff
@@ -80,7 +80,7 @@ describe("#CachedTargetingGetter", () => {
       context,
     });
 
-    assert.equal(stub.callCount, 5);
+    assert.equal(stub.callCount, 9);
```

Comment:
We've been constantly updating this for various patches… I wonder if it makes sense to do something more dynamic, e.g., `assert.equal(stub.callCount, messages.filter(…).length)` ?